src/lineitemstate-csv-parser.js

```diff
@@ -0,0 +1,83 @@
+import { SphereClient } from 'sphere-node-sdk'
+import { userAgent } from 'sphere-node-utils'
+import _ from 'underscore'
+import highland from 'highland'
+import JSONStream from 'JSONStream'
+import csv from 'csv-parser'
+import loadSelfPkg from 'load-pkg'
+import CONS from './constants'
+
+export default class LineItemStateCsvParser {
+  constructor (apiClientConfig, logger, config = {}) {
+    const selfPkg = loadSelfPkg.sync(process.cwd())
```

Comment:
Since it's sync and the `package.json` is always in the root maybe we can just get the version with `import` so we don't need a module for it. Like [on this line](https://github.com/commercetools/csv-parser-orders/pull/7/files#diff-d9d9214113dea1c364abd4ba3c70be11R10).